sdk-fetch

```diff
@@ -1,8 +1,13 @@
 #!/bin/bash
 set -euxo pipefail
+UPSTREAM_SOURCE_FALLBACK="${UPSTREAM_SOURCE_FALLBACK:-'false'}"
 # shellcheck disable=SC2046
-curl --fail --remote-name-all --remote-time \
+if ! curl --fail --remote-name-all --remote-time \
     $(awk -F '[ ()]' '/^SHA512 \(/ {
         printf "https://cache.bottlerocket.aws/%s/%s/%s\n", $3, $6, $3
-    }' "$1")
+    }' "$1") \
+&& [[ "${UPSTREAM_SOURCE_FALLBACK}" == 'true' ]]; then
+    curl --fail --remote-name-all --remote-time --location \
+        $(awk '/^#\s((s)?ftp|http(s)?):\/\// {printf "%s\n", $2}' "$1")
```

Comment:
This could be simplified as `$(awk '/^# http/ {printf "%s\n", $2}' "$1")`, but the committed approach felt more complete.

Reply:
We are still in control of the checksums file right?

Reply:
Right, the checksums are still compared to the `hashes/my-thing` files. The only thing this does different is rather than curl the generated lookaside url it will curl whatever (s)ftp/http(s) urls are in the comment lines of those files.